src/MessageHandler/FeedExportHandler.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Omikron\FactFinder\Shopware6\MessageHandler;
+
+use Omikron\FactFinder\Shopware6\Message\FeedExport;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+class FeedExportHandler implements MessageHandlerInterface
```

Comment:
Please implement handler as its described in Showpare 6 documentation
https://developer.shopware.com/docs/guides/plugins/plugins/framework/message-queue/add-message-handler
Handler should extend `AbstractMessageHandler` and be tagged with `messenger.message_handler`